app/controllers/concerns/agent_logging_concern.rb

```diff
@@ -0,0 +1,17 @@
+module AgentLoggingConcern
+  extend ActiveSupport::Concern
+
+  included do
+    around_action :with_agent_logging_tag
+  end
+
+  private
+
+  def with_agent_logging_tag(&)
+    if current_agent.present?
```

Comment:
Il vaut mieux toujours utiliser le helper `logged_in?` pour éviter les effets de bord
```suggestion
    if logged_in?
```

Reply:
On a pas accès à `logged_in?` dans l'api. C'est acceptable avec `current_agent` non ? Quel est le risque ? ou tu veux que je fasse différemment en fonction de l'app et de l'api ?

Reply:
Le seul risque que je vois c'est si un agent essaie de faire une action sur l'appli (par ex mettre à jour un usager depuis l'appli) mais que la requête est finalement redirigée vers la page d'authentification puisque la session de l'agent n'était plus valide. Ça rendrait le debug un peu plus difficile (il faudrait vérifier que la réponse de l'appel). 
Mais le cas étant un peu niche je pense que c'est ok ?
Sinon on peut aussi rajouter une méthode `logged_in?` côté API 🤷‍♂️